sdk/turing/generated/model/id_object1.py

```diff
@@ -0,0 +1,179 @@
+"""
+    Turing Minimal Openapi Spec for SDK
+
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501
+
+    The version of the OpenAPI document: 0.0.1
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import re  # noqa: F401
+import sys  # noqa: F401
+
+from turing.generated.model_utils import (  # noqa: F401
+    ApiTypeError,
+    ModelComposed,
+    ModelNormal,
+    ModelSimple,
+    cached_property,
+    change_keys_js_to_python,
+    convert_js_args_to_python_args,
+    date,
+    datetime,
+    file_type,
+    none_type,
+    validate_get_composed_info,
+)
+
+
+class IdObject1(ModelSimple):
```

Comment:
Can we get rid of the `IdObject_1` from the openAPI specs and give it a name too ?